### ISSUE TITLE
[Salesforce] `RowExpander` throws error when collapse the record

### ISSUE THREAD

**Issue by chuckn0rris** [bryntum/support#7791]
```
rowExpander: {
                    spanRegions: true,
                    renderer({grid}) {
                          const button = document.createElement('button');
                          button.textContent = 'Close';
                          button.addEventListener('click', event => {
                                  grid.features.rowExpander.collapse(record);
                          });
                          return {
                            tag: 'div',
                            children: [button]
                          };
                    }
                }
```

```
VM3783:89907 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'get')
    at RowExpander.unobserveResize (eval at <anonymous> (eval at <anonymous> (aura_proddebug.js:35725:87)), <anonymous>:89907:74)
    at RowExpander.collapse (eval at <anonymous> (eval at <anonymous> (aura_proddebug.js:35725:87)), <anonymous>:89597:16)
```

[Forum post](https://forum.bryntum.com/viewtopic.php?f=43&t=26795&p=133756#p133756)

Hello,

I was doing some development using the grid’s row-expander feature and noticed an issue when trying to call collapse on the expanded row, whether that be by using a custom button within a template I created or even the out-of-the-box column. I cannot reproduce this in your examples, but could be specific to the Lightning Web Component version of the library since that is where I am seeing this problem. This is also with LWS enabled.

I get the following error when I try and collapse the row using both my custom HTML and the dedicated column when it’s configured (see attached).

Disabling LWS, I’m observing that the dedicated expand/collapse column and custom HTML button is now working, but my calls to the expand function on my column click handlers is not working.

I'm using 5.6.0.

To summarize:

With LWS enabled, collapsing an expanded row with both the dedicated column and programmatic calls to collapse are not working.
With LWS disabled, the dedicated column works to expand and collapse the row. The HTML button inside my expanded row can collapse the template, but programmatic calls to expand are not working.
![row expander collapse error.png](https://forum.bryntum.com/download/file.php?id=24833)
